soc/arm/nordic_nrf/nrf53/Kconfig.defconfig.nrf5340_CPUAPP_QKAA

```diff
@@ -11,4 +11,12 @@ config SOC
 config NUM_IRQS
 	default 69
 
+config NRF5340_CPUAPP_ERRATUM19
+	bool "Include workarounds for Erratum 19"
+	default y
+	help
+	  This MUST be enabled if there is a chance the code will be run on
+	  nRF5340 Engineering A (Variant AAAB).
```

Comment:
Reuse text from the Errata text: "nRF5340 Engineering A, build codes QKAA-AB0"

Reply:
Done